x-pack/plugins/apm/public/components/shared/managed_table/index.tsx

```diff
@@ -128,8 +130,9 @@ function UnoptimizedManagedTable<T>(props: Props<T>) {
       totalItemCount: items.length,
       pageIndex: page,
       pageSize,
+      pageSizeOptions,
```

Comment:
it seems that we use `ManagedTable` in many places to display the table. Why don't you define the default values here for `pageSizeOptions={[10, 25, 50]}` instead of each component that uses `ManagedTable`